pac4j-core/src/main/java/org/pac4j/core/context/HttpConstants.java

```diff
@@ -42,6 +42,8 @@
 
     String AUTHORIZATION_HEADER = "Authorization";
 
+    String BASIC_HEADER_PREFIX = "Basic ";
```

Comment:
According to https://tools.ietf.org/html/rfc2617, it's "Basic ", but I think we could use a `equalsIgnoreCase` in the `BasicAuthExtractor`. No one has ever reported an issue on this...